CMakeLists.txt

```diff
@@ -1539,68 +1540,38 @@ IF(ALSOFT_EXAMPLES)
 
     IF(SDL2_FOUND)
         IF(SDL_SOUND_FOUND)
+            TARGET_INCLUDE_DIRECTORIES(ex-common
+                PUBLIC
+                    ${SDL2_INCLUDE_DIR}
+                    ${SDL_SOUND_INCLUDE_DIR}
+            )
+            TARGET_LINK_LIBRARIES(ex-common
+                PUBLIC
+                    ${SDL_SOUND_LIBRARIES}
+                    ${SDL2_LIBRARY}
+                    ${MATH_LIB}
+            )
```

Comment:
Does this work as seemingly intended, that it only applies to the future uses of `ex-common` and not retroactively?

Reply:
No, it applies it retroactively AFAIK. It means the example "alrecord" will linked with the SDL unnecessary.

These are just examples, so I guessed it  isn't really an issue. If it is an issue, I can skip this patch. What do you think?

Reply:
You could probably instead create a new target (e.g. `ex-common-sdl`) that includes ex-common and those additional SDL flags, then use that for the examples that need SDL. Otherwise, it would be better to skip the patch so examples don't link against SDL and ffmpeg unnecessarily.

Reply:
I skipped patch #1.